libs/vkd3d-shader/spirv.c

```diff
@@ -11813,6 +11827,56 @@ void vkd3d_shader_extract_feature_meta(struct vkd3d_shader_code *code)
                 meta |= VKD3D_SHADER_META_FLAG_EMITS_TRIANGLES;
 
         }
+        else if ((op == SpvOpDecorate && count == 4) ||
+                (op == SpvOpMemberDecorate && count == 5))
```

Comment:
Member decorate? SpvBuiltInSampleMask cannot be placed in IO block.

Reply:
Is the intention to be more extensible here?

Reply:
Pretty much, yes.